contracts/src/v0.8/feeds/DualAggregator.sol

```diff
@@ -1,88 +1,78 @@
 
-  /**
-   *
-   * Section: Variables used in multiple other sections
-   *
-   */
+  // ================================================================
+  // │            Variables used in multiple other sections         │
+  // ================================================================
+
   struct Transmitter {
-    bool active;
-    // Index of oracle in s_signersList/s_transmittersList
-    uint8 index;
-    // juels-denominated payment for transmitters, covering gas costs incurred
-    // by the transmitter plus additional rewards. The entire LINK supply (1e9
-    // LINK = 1e27 Juels) will always fit into a uint96.
-    uint96 paymentJuels;
+    bool active; // ─────────╮ True if active.
+    uint8 index; //          │ Index in `s_transmittersList`.
+    //                       │
+    //                       │ Juels-denominated payment for transmitters, covering gas costs incurred
+    //                       │ by the transmitter plus additional rewards. The entire LINK supply (1e9
+    uint96 paymentJuels; // ─╯ LINK = 1e27 Juels) will always fit into a uint96.
   }
 
-  mapping(address /* transmitter address */ => Transmitter) internal s_transmitters;
+  mapping(address transmitter => Transmitter) internal s_transmitters;
 
   struct Signer {
-    bool active;
-    // Index of oracle in s_signersList/s_transmittersList
-    uint8 index;
+    bool active; // ─╮ True if active.
+    uint8 index; // ─╯ Index of oracle in s_signersList.
   }
 
-  mapping(address /* signer address */ => Signer) internal s_signers;
+  mapping(address signer => Signer) internal s_signers;
 
-  // s_signersList contains the signing address of each oracle
+  /// @notice s_signersList contains the signing address of each oracle
   address[] internal s_signersList;
 
-  // s_transmittersList contains the transmission address of each oracle,
-  // i.e. the address the oracle actually sends transactions to the contract from
+  /// @notice s_transmittersList contains the transmission address of each oracle,
+  /// i.e. the address the oracle actually sends transactions to the contract from
   address[] internal s_transmittersList;
 
-  // We assume that all oracles contribute observations to all rounds. this
-  // variable tracks (per-oracle) from what round an oracle should be rewarded,
-  // i.e. the oracle gets (latestAggregatorRoundId -
-  // rewardFromAggregatorRoundId) * reward
+  /// @notice We assume that all oracles contribute observations to all rounds. this
+  /// variable tracks (per-oracle) from what round an oracle should be rewarded,
+  /// i.e. the oracle gets (latestAggregatorRoundId - rewardFromAggregatorRoundId) * reward
   uint32[MAX_NUM_ORACLES] internal s_rewardFromAggregatorRoundId;
 
   bytes32 internal s_latestConfigDigest;
 
   // Storing these fields used on the hot path in a HotVars variable reduces the
-  // retrieval of all of them to a single SLOAD.
+  // retrieval of all of them to two SLOADs.
```

Comment:
This changed to two SLOADs because we added the `uint32 latestSecondaryRoundId`, maybe we could decrease the size of some parameters.

E.g. `uint16 accountingGas` should be enough (?)

Reply:
If you can get within 32 bytes by decreasing types that would be ideal!

Reply:
Yes, this would be fantastic, I'll be investigating the parameters size.